content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md

```diff
@@ -167,6 +167,13 @@ resources:
     memory: 128Mi
 ```
 
+{{< note >}}
+
+`LimitRange` does not check the consistency of default values it applied. Default value for the limit that is set by `LimitRange` may be less than the request value specified by Container which will lead to Pod not being scheduleable. See ["Overview of LimitRange"](/docs/concepts/policy/limit-range/#overview-of-limit-range) for more details.
```

Comment:
A `LimitRange` does **not** check the consistency the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be scheduleable.
```
Is that correct? My suspicion is that the final Pod isn't even able to pass admission validation.

It's OK to link to the introductory text; I've omitted it here because in another part of this review, I suggested a different heading.